slave/buildslave/monkeypatches/bug4881.py

```diff
@@ -189,7 +191,7 @@ def _resourceFDImplementation(self):
         # to close
         if maxfds > 1024:
             maxfds = 1024
-        return xrange(maxfds)
+        return list(range(maxfds))
```

Comment:
When using python future, range will return an iterator in 2.7.
you can read more about it in the [documentation](http://python-future.org/compatible_idioms.html#xrange) for python future